data/events/2024/brasilia/main.yml

```
@@ -0,0 +1,149 @@
+
+cfp_open: "false"
+#cfp_link: "" #if you have a custom link for submitting proposals, add it here. This will control the Propose menu item as well as the "Propose" button.
+
+registration_date_start: "" #2022-10-01T00:09:00-03:00 # start accepting registration. Leave blank if registration is not open yet
+registration_date_end: "" #2022-10-31T00:09:00-03:00 # close registration. Leave blank if registration is not open yet.
+
+registration_closed: "" #set this to true if you need to manually close registration before your registration end date
+registration_link: "" # If you have a custom registration link, enter it here. This will control the Registration menu item as well as the "Register" button.
+
+# Location
+#
+coordinates: "" # The coordinates of your city. Get Latitude and Longitude of a Point: http://itouchmap.com/latlong.html
+location: "TBA" # Defaults to city, but you can make it the venue name.
+location_address: "" #Optional - use the street address of your venue. This will show up on the welcome page if set.
+
+nav_elements: # List of pages you want to show up in the navigation of your page.
+  #   icon: "" deprecated - This is a font-awesome icon that will display on small screens. Choose at http://fontawesome.io/icons/
+  - name: local
+    url: /events/2024-brasilia/local
+  #  - name: inscricao
+  #    url: ""
+  - name: código de conduta
+    url: /events/2024-brasilia/conduct
+  - name: contato
+    url: /events/2024-brasilia/contact
+  - name: Programação
+    url: /events/2024-brasilia/program
+
+# These are the same people you have on the mailing list and Slack channel.
+team_members: # Name is the only required field for team members.
+  - name: "Ruan Oliveira"
+    employer: ""
+    image: ""
+    bio: ""
+    linkedin: ""
+
+  - name: "Jorge Santana"
+    employer: ""
+    image: ""
+    twitter: ""
+    bio: ""
+    linkedin: "https://www.linkedin.com/in/jorgesantanadeveloper"
+
+  - name: "Herberson Miranda"
+    employer: ""
+    image: ""
+    bio: ""
+    linkedin: ""
+
+  - name: "Jady Pâmella"
+    employer: "BRB"
+    image: ""
+    bio: ""
+    linkedin: ""
+
+  # - name: "Marília Angela"
+  #   employer: "Itaú"
+  #   image: "marilia-angela.jpg"
+  #   bio: ""
+  #   linkedin: "https://www.linkedin.com/in/mariliaangela/"
+
+  # - name: "Shenia Andrade"
+  #   employer: "Neon"
+  #   image: "shenia-andrade.jpg"
+  #   bio: ""
+  #   linkedin: "https://www.linkedin.com/in/shenia-tamires-andrade/"
+
+  # - name: "Yago Nobre"
+  #   employer: "Nubank"
+  #   image: "yago-nobre.jpg"
+  #   twitter: "yagoyns"
+  #   bio: ""
+  #   linkedin: "https://www.linkedin.com/in/yagonobre"
+
+organizer_email: "brasilia@devopsdays.org" # Put your organizer email address here
+
+# List all of your sponsors here along with what level of sponsorship they have.
+# Check data/sponsors/ to use sponsors already added by others.
+sponsors_accepted: "yes" # Whether you want "Become a XXX Sponsor!" link
```

Are you actually ready to accept sponsors? It doesn't appear to be the case right now (no prospectus, default sponsor page, etc).
```suggestion
sponsors_accepted: "no" # Whether you want "Become a XXX Sponsor!" link
```